apps/web/scripts/kernel/handlers/solanaHandler.ts

```diff
@@ -14,7 +15,7 @@ import { triggerActionToGetPrivateKey } from '../utils/handler';
 import { getPrivateKey, settings } from './../utils/handler';
 
 export const handleSignTransaction: MessengerCallback = async (
-	payload,
+	payload: MessagePayload,
```

Comment:
remove this, already defined in `MessengerCallback`.

Reply:
Ok boss!